src/MessagePack.UnityClient/Assets/Scripts/MessagePack/Resolvers/CompositeResolver.cs

```diff
@@ -38,124 +30,102 @@ public CompositeResolver()
         /// The composite resolver that includes the AOT code generated resolver and several standard ones
         /// that the AOT code generator expects to be present.
         /// </returns>
-        public static CompositeResolver CreateForAot(IFormatterResolver aotResolver)
+        public static IFormatterResolver CreateForAot(IFormatterResolver aotResolver)
```

Comment:
I don't know why exists CreateForAot, remove it.
AOT can use some reflection methods.

Reply:
> I don't know why exists CreateForAot, remove it.

I did this because in your docs you describe how an AOT generated resolver should be linked with several others in order to work at runtime. I expect the several others is a fixed set that your codegen system assume will be present. Today, the user has to read the docs to find out what that set is and hard-code them in their app themselves. Yet AOT seems like a premium enough feature of this library to have a method that makes it all that much easier. Then if the set of resolvers change in this library over time and the AOT codegen needs more or less resolvers to be included, we can change the library and keep the app working.

> AOT can use some reflection methods.

I'm not sure what you mean by this. Can you elaborate.

Reply:
I've removed the method and updated the docs that referenced it. Can you confirm that the doc's guidance is what you had in mind?